tutorials/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -465,6 +465,43 @@ GDScript               C#
 ``length_squared()``   ``LengthSquared``
 =====================  =============================================================
 
+Color
+-----
+
+Structs cannot have parameterless constructors in C#. Therefore, ``new Color()``
+initializes all primitive members to their default value (which represents the transparent black color).
+Please use ``Colors.Black`` for the equivalent of ``Color()`` in GDScript and C++.
+
+The global ``Color8`` method to construct a Color from bytes is available as a static method
+in the Color type.
+
+The Color constants are available in the ``Colors`` static class as readonly properties.
+
+The following method was converted to a property with a different name:
+
+====================  ==============================================================
+GDScript              C#
+====================  ==============================================================
+``get_luminance()``    ``Luminance``
+====================  ==============================================================
+
+The following method was converted to a method with a different name:
+
+====================  ==============================================================
+GDScript              C#
+====================  ==============================================================
+``html()``             ``FromHTML``
```

Comment:
The C# method is missing parentheses, and we could probably add the argument type to both GDScript and C# method here.

Reply:
Oh, yeah, my bad.